File: public/phase-2-20160620022203/JAVASCRIPT/flowers-girls-and-grandma.js
// Name : Flowers Girls and Grandma
// input:
// n :  the number of girls
// p : Grandmas favourite prime number
// lr : each row corresponds to the range of flowers that each girl can grow
var idx,result;
var elements = readline().split(" ");
var n = parseInt(elements[0], 10);
var p = parseInt(elements[1], 10);
var lr = new Array([]);
var lr_elements = "";
for(idx=0;idx<n;idx++){
	lr_elements = readline().split(' ');
	lr[idx] = new Array([]);
	for(jdx=0;jdx<2;jdx++){
		lr[idx][jdx] = parseInt(lr_elements[jdx], 10);
	}
}


// write your code here
// store your results in `result`

// Dummy Data
result = 0.0;


// output
print(result);

File: public/phase-2-20160620022203/JAVA/flowers-girls-and-grandma.java
// Name : Flowers Girls and Grandma
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n :  the number of girls
		// p : Grandmas favourite prime number
		// lr : each row corresponds to the range of flowers that each girl can grow
		int idx;
		int jdx;
		Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int n = Integer.parseInt(elements[0]);
		int p = Integer.parseInt(elements[1]);
		int[][] lr = new int[n][2];
		String[] lr_elements = new String[100];
		for(idx=0;idx<n;idx++){
			lr_elements = (in.nextLine()).split(" ");
			for(jdx=0;jdx<2;jdx++){
				lr[idx][jdx] = Integer.parseInt(lr_elements[jdx]);
			}
		}


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		float result=0.0;

		System.out.println(result);

	}
}